### PR TITLE
Example vocabulary cards

### PR DESCRIPTION
**Pull request by annadragan**
First Issue of my Project: I have created some example Vocabulary Cards with the data from my example Array
[Issue 1:](https://github.com/annadragan/capstone-project/issues/9)

### CODE REVIEW

File: src/components/CardList.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Card from './Card';
+import styled from 'styled-components';
+import { vocabulary } from '../db';
+
+export default function CardList() {
+  return (
+    <>
+      <Wrapper>
+        <AllCards role="list" aria-label="vocabulary cards" rows={3}>
```
Comment (Sven-Lyco) [inline src/components/CardList.js:9]: Maybe you can choose a better name for the list here. For example `StyledList`

File: src/components/CardList.spec.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import CardList from './CardList';
+
+describe('CardList', () => {
+  it('Render a list of vocabulary cards', () => {
```
Comment (Sven-Lyco) [inline src/components/CardList.spec.js:5]: You can write here `it('renders a list of vocabulary cards',()...` 🙂  like in your `Card.spec.js`

File: src/components/Card.spec.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import Card from './Card';
+import { render, screen } from '@testing-library/react';
+
+describe('Card', () => {
+  it('renders a word and example and/or explanation', () => {
```
Comment (Sven-Lyco) [inline src/components/Card.spec.js:5]: I would suggest you remove the part `/or`, because your testing the explanation 🙂

**Review by Sven-Lyco — COMMENTED** [review]
You've done some great work here! Keep it up 🚀

File: src/components/CardList.js (hunk 1 above)
Comment (annadragan) [reply src/components/CardList.js:9]: right, it sounds better :)

File: src/components/Card.spec.js (hunk 3 above)
Comment (annadragan) [reply src/components/Card.spec.js:5]: thank you!

**Pull request merged**